Content.Client/Chat/MarkupTags/ColorValueContentTag.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Diagnostics;
+using Content.Shared.Chat.ContentMarkupTags;
+using Robust.Shared.Utility;
+
+namespace Content.Client.Chat.MarkupTags;
+
+public sealed class ColorValueContentTag : IContentMarkupTag
+{
+    public string Name => "ColorValue";
+
+    // TODO: These values should probably be retrieved via yaml, and be customizable in options! This solution works for now!
+    private Dictionary<string, Color> _colors = new Dictionary<string, Color>()
+    {
+        ["Base"] = Color.White,
+        ["Speech"] = Color.LightGray,
+        ["Emote"] = Color.LightGray,
+        ["Whisper"] = Color.DarkGray,
+        ["LOOC"] = Color.MediumTurquoise,
+        ["OOC"] = Color.LightSkyBlue,
+        ["Dead"] = Color.MediumPurple,
+        ["IngameAnnouncement"] = Color.Yellow,
+        ["IngameAlert"] = Color.Red,
+        ["Server"] = Color.Orange,
+        ["AdminChat"] = Color.HotPink,
+        ["AdminAlert"] = Color.Red,
+        ["Radio.Common"] = Color.FromHex("#2cdb2c"),
+        ["Radio.Security"] = Color.FromHex("#ff4242"),
+        ["Radio.Supply"] = Color.FromHex("#b48b57"),
+        ["Radio.Command"] = Color.FromHex("#fcdf03"),
+        ["Radio.Service"] = Color.FromHex("#539c00"),
+        ["Radio.Science"] = Color.FromHex("#cd7ccd"),
+        ["Radio.Engineering"] = Color.FromHex("#ff733c"),
+        ["Radio.Medical"] = Color.FromHex("#57b8f0"),
+        ["Radio.Syndicate"] = Color.FromHex("#8f4a4b"),
+        ["Radio.Freelance"] = Color.FromHex("#f6ce64"),
+        ["Radio.Service"] = Color.FromHex("#539c00"),
+        ["Radio.Binary"] = Color.FromHex("#2ed2fd"),
+        ["Radio.CentCom"] = Color.FromHex("#2681a5"),
+        ["Radio.Handheld"] = Color.FromHex("#967101"),
+    };
+
+    public List<MarkupNode>? OpenerProcessing(MarkupNode node)
+    {
+        return new List<MarkupNode>() { new MarkupNode("color", new MarkupParameter(ColorValueLookup(node)), null) };
```

Comment:
i'm a little bit afraid of perf for heavily loaded server with all thouse lists - i'm not agitating to do some pre-emptive optimizations but it might be interesting to make all chat code allocation-less... maybe later!